allennlp/models/heads/classifier_head.py

```diff
@@ -0,0 +1,136 @@
+from typing import Dict, Optional
+
+from overrides import overrides
+import torch
+
+from allennlp.data import Vocabulary
+from allennlp.models.heads.head import Head
+from allennlp.models.model import Model
+from allennlp.modules import FeedForward, Seq2VecEncoder
+from allennlp.modules.seq2vec_encoders import ClsPooler
+from allennlp.training.metrics import CategoricalAccuracy
+
+
+@Head.register("classifier")
+class ClassifierHead(Model):
```

Comment:
Usually a class gets registered under its own super type, but not here. I guess it does not matter?

Reply:
Now that I've read the definition of `Head`, it seems to me that this should derive from `Head`.

Reply:
Thanks, this was a copy-paste error.